docs/maps/regionmap.rst

```diff
@@ -433,11 +433,15 @@ Similarly, the map contents can also be plotted as a histogram:
 Writing and reading a RegionNDMap to/from a FITS file
 -----------------------------------------------------
 Region maps can be written to and read from a FITS file with the
-`~RegionNDMap.write()` and `~RegionNDMap.read()` methods. The
-format specification depends on which type of axis the region has:
- 
-Region maps with a reconstructed energy axis
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+`~RegionNDMap.write()` and `~RegionNDMap.read()` methods. Currently
+the following formats are supported:
+
+- "gadf": a generic serialisation format with support for ND axes
```

Comment:
See my comment below...

Reply:
Choosing a different name for this format is possible but introduces the problem of defining multiple formats in `MapDataset.write()`. Consider the case where `dataset.psf` is a `RegionGeom` (assuming we want to support this)  and one uses `MapDataset.write(format="gadf")`. I guess we don't really want to introduce something like `,write(format={"counts": "gadf", "background": "fgst-ccube", "psf": "gp-region"})`, this would make the number of possible single fits file definitions for a `MapDataset` explode...